### PR TITLE
Add Terraform Destroy pipeline for OMOP environment

### LINKED ISSUE #44: Investigate Adding Terraform Destroy for Environment Pipeline
* Investigate automating TF destroy option for TF Environment Pipeline

### PR DESCRIPTION
**Pull request by yradsmikham**
Close #44 

TF Destroy Pipeline: https://dev.azure.com/US-HLS-AppInnovations/yvonne-test-OHDSIonAzure/_build?definitionId=241

### CODE REVIEW

File: infra/terraform/bootstrap/azure_devops.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ resource "azuredevops_project" "project" {
   work_item_template = "Agile"
 
   lifecycle {
-    prevent_destroy = true # prevent destroying the project
+    #prevent_destroy = true # prevent destroying the project
```
Comment (plooploops) [inline infra/terraform/bootstrap/azure_devops.tf:16]: Just checking if we want to have this commented out in the bootstrap

File: infra/terraform/bootstrap/azure_devops.tf [hunk 2]
```diff
@@ -38,7 +38,7 @@ resource "azuredevops_git_repository" "repo" {
   # }
   # Use Terraform import instead, otherwise this resource will destroy the existing repository.
   lifecycle {
-    prevent_destroy = true # prevent destroying the repo
+    #prevent_destroy = true # prevent destroying the repo
```
Comment (plooploops) [inline infra/terraform/bootstrap/azure_devops.tf:41]: Same here

File: infra/terraform/bootstrap/azure_devops.tf (hunk 1 above)
Comment (yradsmikham) [reply infra/terraform/bootstrap/azure_devops.tf:16]: oh yes, commented on accident! Thank you!

**Review by plooploops — APPROVED** [review]
Looks good!

We'll have a docs issue to cover changes for the new pipeline

**Pull request merged**